docs/dev/project_management.rst

```diff
@@ -8,16 +8,52 @@ management tools as well as `Zenhub <https://www.zenhub.com>`__ which provides
 additional features for sprint planning, task estimation, and progress reports.
 
 -------------------------------------------------------------------------------
-Issues and Project Tracking
+Agile Scrum Kanban Iteration SDLC Process
 -------------------------------------------------------------------------------
-We use `Github issues <https://github.com/catalyst-cooperative/pudl/issues>`__ to
-track bugs, enhancements, support requests, and just about any other work that goes
-into the project. Try to make sure that issues have informative tags so we can find
-them easily.
-
-We use Zenhub Sprints, Epics, and Releases to track our progress. These won't be
-visible unless you have the `ZenHub browser extension
-<https://www.zenhub.com/extension>`__ installed.
+
+We merge code to ``dev`` continuously, but plan work in two-week sprints.
+
+We track work as items (GitHub issues and PRs) in a Catalyst-wide Github Project.
+
+During each sprint, people on each team pull from a team-specific list of
+prioritized & scoped items.
+
+We have 3 meetings per sprint:
+
+- planning meeting: overall goal is "get a good idea of what work is coming up
+  in the sprint"
+
+  - mark items with a new iteration
+
+    - enough that that we feel like there's enough work to do
```

Comment:
Maybe someday we can use "velocity" to achieve this magic balance but for now I think "vibe" is what we've got.

Reply:
Ah I totally forgot about the Meetings section of catalyst_101.md - just looked for 'project management' and found this file. Yeah, "iteration" is the GH term for sprint but maybe I can just rename it in the project. Though we definitely don't do capital-S Sprints here so 🤷 maybe 'iteration' or 'cycle' is more accurate.

(side note: it would be cool if our documentation all lived in one place but that's definitely not in scope)